### PR TITLE
fix: fix image color on gmail dark mode

### PR DESCRIPTION
**Pull request by wa0x6e**
## 🧿 Current issues / What's wrong ?

Gmail on mobile does not honor the CSS color scheme media query for the dark mode, and is using its own method to create a dark mode for the emails.

Screenshot_20230629_055246_Gmail

## 💊 Fixes / Solution

Edit the image, and force a white background on the logo with black text. 

## 🚧 Changes

- Use an image with a white background, instead of transparent one.

## 🛠️ Tests

- Use the email previewer.
- The logo on light theme should have a white background 

To really test on gmail itself, we have to send an email from the prod server, as local testing sending emails will not have logo (since they're from localhost)

### CODE REVIEW

File: src/templates/partials/layout.html.hbs [hunk 1]
```diff
@@ -39,7 +39,7 @@
             <tbody>
               <tr>
                 <td>
-                  <img alt="Snapshot logo" class="logo-light" src="{{host}}/images/logo.png" />
```
Comment (Todmy) [inline src/templates/partials/layout.html.hbs:42]: do we need `logo.png`?

**Review by Todmy — COMMENTED** [review]
t

File: src/templates/partials/layout.html.hbs (hunk 1 above)
Comment (wa0x6e) [reply src/templates/partials/layout.html.hbs:42]: Kept there in case we need to generate another variant

**Review by Todmy — APPROVED** [review]
tACK

**Pull request merged**